pyomo/common/tests/test_deprecated.py

```diff
@@ -13,6 +13,26 @@
 class TestDeprecated(unittest.TestCase):
     """Tests for deprecated function decorator."""
 
+    def test_deprecation_warning(self):
+        DEP_OUT = StringIO()
+        with LoggingIntercept(DEP_OUT, 'pyomo.core'):
+            deprecation_warning(None, version='1.2', remove_in='3.4')
+
+        self.assertIn('DEPRECATED: This has been deprecated',
+                      DEP_OUT.getvalue())
+        self.assertIn('(deprecated in 1.2, will be removed in 3.4)',
+                      DEP_OUT.getvalue().replace('\n',' '))
```

Comment:
Why do we need the `replace()` here?

Reply:
The message gets line wrapped and I felt it was easier to remove the line breaks and eliminate the possibility that changes in the column width would lead to spurious test failures.